lib/twurl/cli.rb

```diff
@@ -92,15 +90,15 @@ def parse_options(args)
           end
         end
 
-        arguments                 = option_parser.parse!(args)
-        Twurl.options.command     = extract_command!(arguments)
-        Twurl.options.path        = extract_path!(arguments)
```

Comment:
```rb
        arguments                 = option_parser.parse!(args)
        Twurl.options.command     = extract_command!(arguments)
        Twurl.options.path        = extract_path!(arguments)
        Twurl.options.subcommands = arguments
```

Because of the below comment, I'd rather keep these lines as is, and move `Twurl.options.subcommands = arguments` to here instead. Does this make sense to you?